src/main/com/smithsmodding/armory/client/model/loaders/ArmorComponentModelLoader.java

```diff
@@ -33,10 +30,6 @@ public boolean accepts(@Nonnull ResourceLocation modelLocation) {
 
     @Override
     public IModel loadModel(@Nonnull ResourceLocation modelLocation) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)